src/Strategies/DNSRecord.php

```diff
@@ -16,6 +18,23 @@ class DNSRecord implements StrategyInterface
      */
     public function verify(string $url, DomainVerifiableInterface $domainVerifiable)
     {
+        $dns = new Dns($url);
+        $txtRecords = $dns->getRecords('TXT');
+        $verificationName = config('domain_verifier.verification_name');
+        $txtRecords = explode(PHP_EOL, $txtRecords);
+        $domainToken = array();
+
+        foreach ($txtRecords as $item) {
+            if (strpos($item, $verificationName)) {
+                array_push($domainToken, substr($item, strpos($verificationName, '=') + 2, -1));
+            }
+        }
+
+        $verificationToken = DomainVerificationFacade::getTokenFor($url, $domainVerifiable)->token;
```

Comment:
I think we only `new DNS` strategy. Call `verify` method then assert the result.